telemetry/src/state.rs

```diff
@@ -56,6 +58,32 @@ pub struct TelemetryState {
 
     /// Number of cache hits for queries emitted by the relayer, per chain and query type
     query_cache_hits: Counter<u64>,
+
+    /// Number of time the relayer had to reconnect to the WebSocket endpoint, per chain
+    ws_reconnect: Counter<u64>,
+
+    /// How many IBC events did Hermes receive via the WebSocket subscription, per chain
+    ws_events: Counter<u64>,
+
+    /// How many messages Hermes submitted to the chain, per chain
+    msg_num: Counter<u64>,
+
+    /// The balance in each wallet that Hermes is using, per wallet, denom and chain
+    wallet_balance: ValueRecorder<u64>,
```

Comment:
the `ValueRecorder` using `HistogramAggregator` ,   so I don't think it applies to `wallet_balances`

Reply:
Good point, what metric type would you use instead?

Reply:
I guess we need a custom `AggregatorSelector` for the Prometheus exporter.

Reply:
> I guess we need a custom `AggregatorSelector` for the Prometheus exporter.

agree!
https://docs.rs/opentelemetry/latest/src/opentelemetry/sdk/metrics/selectors/simple.rs.html#29-61

Reply:
> Good point, what metric type would you use instead?

I am also try to using `Prometheus::gauge` for instead, but it is not so convenient. Label must be specified when creating.
code like that: 
```rust
// create 
let registry = prometheus::default_registry();
let opts = Opts::new("signer_balance", "Balance Of Signer")
                .const_label("chain_id", chain_id.to_string())
                .const_label("address", address);
let gauge = Gauge::with_opts(opts).unwrap();
registry.register(Box::new(gauge)).unwrap();

// use 
gauge.set(balance);
```

Reply:
Done https://github.com/informalsystems/ibc-rs/pull/2182/commits/800f73b79a086da95cf4ec293e774a6b0a28c0c3